composer.json

```diff
@@ -21,8 +21,8 @@
     },
     "require": {
         "php": ">=5.3.0",
-        "symfony/filesystem": "~2.3",
-        "symfony/process": "~2.3"
+        "symfony/filesystem": "~3.1",
+        "symfony/process": "~3.1"
```

Comment:
Could you try to change these version constraints to `>=2.3.0 < 4` and check if everything keeps working?